apps/meteor/ee/app/livechat-enterprise/server/lib/Helper.js

```diff
@@ -236,7 +237,7 @@ export const updateInquiryQueuePriority = (roomId, priority) => {
 
 export const removePriorityFromRooms = (priorityId) => {
 	LivechatRooms.findOpenByPriorityId(priorityId).forEach((room) => {
-		updateInquiryQueuePriority(room._id);
+		Promise.await(updateInquiryQueuePriority(room._id));
```

Comment:
If syncronization isn't required, then maybe we can benifit from Promise.all() here